iothub/device/src/MessageAcknowledgementType.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using static System.Net.WebRequestMethods;
+
+namespace Microsoft.Azure.Devices.Client
+{
+    /// <summary>
+    /// The options for acknowledging a cloud to device message.
+    /// </summary>
+    /// <seealso href="https://learn.microsoft.com/azure/iot-hub/iot-hub-devguide-messages-c2d#the-cloud-to-device-message-life-cycle"/>
+    public enum MessageAcknowledgementType
```

Comment:
Does anyone feel this is still too generic sounding? Should I go with ```CloudToDeviceMessageAcknowledgementType``` instead?

Reply:
I commented independently that I think we could drop the word "Type", but I wouldn't be against CloudToDeviceMessageAcknowledgment. It is a mouthful, though.

Reply:
Let's go with ```CloudToDeviceMessageAcknowledgment``` then. Drop the "type" to spare the length, but still sounds descriptive enough